File: src/main/java/kurenai/mybot/handler/impl/AntiMiniAppHandlerProperties.java
package kurenai.mybot.handler.impl;

import kurenai.mybot.handler.HandlerProperties;
import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;

import java.util.List;

@Getter
@Setter
@ConfigurationProperties(prefix = "bot.handler.anti-mini-app")
public class AntiMiniAppHandlerProperties extends HandlerProperties {

    private List<Long> ignoreGroup;
}

File: src/main/kotlin/kurenai/mybot/handler/config/AntiMiniAppHandlerProperties.kt
package kurenai.mybot.handler.config

import kurenai.mybot.handler.HandlerProperties
import org.springframework.boot.context.properties.ConfigurationProperties
import java.util.*

@ConfigurationProperties(prefix = "bot.handler.anti-mini-app")
class AntiMiniAppHandlerProperties : HandlerProperties() {
    var ignoreGroup: List<Long> = Collections.emptyList()
}